lib/storage/providers/IDBKeyVal.js

```diff
@@ -0,0 +1,111 @@
+import {
+    set,
+    keys,
+    getMany,
+    setMany,
+    get,
+    clear,
+    del,
+    delMany,
+    createStore,
+    promisifyRequest,
+} from 'idb-keyval';
+import _ from 'underscore';
+import fastMerge from '../../fastMerge';
+
+// Same config as localforage, so we can swap the providers easily
+const customStore = createStore('OnyxDB', 'keyvaluepairs');
```

Comment:
**confirming:** This will have the existing data from the current DB previously set up by `localforage`?

Reply:
Yes exactly 😊 IT can just be replaced in place